Archipelago.HollowKnight/ConnectionDetails.cs

```diff
@@ -7,5 +7,8 @@ public record ConnectionDetails
         public string SlotName { get; set; }
         public string ServerPassword { get; set; }
         public int ItemIndex { get; set; }
+
+        public string RoomSeed { get; set; }
```

Comment:
This doesn't appear to be used.

Reply:
It's set and checked in StartOrResumeGame() as part of `ApSettings`